### DISCUSSION TITLE
Installation error

### DISCUSSION THREAD

**Discussion by sdb7** [nxrighthere/UnrealCLR#241] [Help] [answered]
![image](https://user-images.githubusercontent.com/6058513/153142954-8d7747b6-1b2c-488f-82d6-bbdcfa09607d.png)

Is there anything i missed, had no problem using 1.30?. 

Start using 1.31 today and that error raised when try to install, I had VSTO 2019 & 2022 Installed and all that Net Core SDK requirement,install at unreal engine Version 4.26.2 on Preexisting Unreal Blueprint project and when i run the ue4 project through editor a warning raised that UnrealCLR uninitialized 

UE4 Editor Log :

![image](https://user-images.githubusercontent.com/6058513/153144735-50eb7462-7826-491b-bb3b-92c0934b25d7.png)
![image](https://user-images.githubusercontent.com/6058513/153145009-d2b7af37-cac1-4897-b9fe-e226f1f08564.png)

**Comment by sdb7** [comment]
and I am sorry if my english is confusing

**Comment by nxrighthere** [comment, marked as answer]
Please, download the latest version from the master branch.
